openedx/features/edly/utils.py

```diff
@@ -134,11 +131,32 @@ def _handle_handout_changes(xblock, old_content):
         new_content_with_replaced_static_urls,
         absolute_urls_of_new_data)
 
-    email_params = get_email_params(xblock)
-    email_params['old_content'] = old_content_with_absolute_urls
-    email_params['new_content'] = new_content_with_absolute_urls
-    students = get_course_enrollments(xblock.location.course_key)
-    send_bulk_mail_to_students.delay(students, email_params, 'handout_changes')
+    old_content_lines = []
+    if old_content_with_absolute_urls:
+        old_content_lines = old_content_with_absolute_urls.split('\n')
+    old_content_lines = [line.strip() for line in old_content_lines]
+    new_content_lines = new_content_with_absolute_urls.split('\n')
+    new_content_lines = [line.strip() for line in new_content_lines]
+    if is_not_deleted(old_content_lines, new_content_lines):
```

Comment:
I don't think we should do this, If all old handouts are deleted and a few new ones are added, users will not be notified. and I don't think we need that. We should compare every new handout with new ones individually

Reply:
Done